packages/edit-site/src/components/left-sidebar/navigation-panel/constants.js

```diff
@@ -1,53 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
 export const TEMPLATES_DEFAULT_DETAILS = {
 	// General
 	'front-page': {
-		title: 'Front page',
-		description: '',
+		title: _x( 'Front Page', 'template name' ),
+		description: __(
+			'Front page template, for both a static page or blog posts index'
+		),
 	},
 	archive: {
-		title: 'Archive',
-		description:
-			'Displays the content lists when no other template is found',
-	},
-	single: {
-		title: 'Single',
-		description: 'Displays the content of a single post',
+		title: _x( 'Archive', 'template name' ),
+		description: __( 'Generic archive template' ),
 	},
 	singular: {
-		title: 'Singular',
-		description: 'Displays the content of a single page',
+		title: _x( 'Singular', 'template name' ),
+		description: __( 'Default template for both single posts and pages' ),
 	},
 	index: {
-		title: 'Default (index)',
-		description: 'Displays the content of a single page',
+		title: _x( 'Index', 'template name' ),
+		description: __( 'Default template' ),
 	},
 	search: {
-		title: 'Search results',
-		description: '',
+		title: _x( 'Search Results', 'template name' ),
+		description: __( 'Search results template' ),
 	},
 	'404': {
-		title: '404',
-		description: 'Displayed when a non-existing page requested',
+		title: _x( '404 (Not Found)', 'template name' ),
+		description: __( 'Template for "not found" errors' ),
 	},
 
 	// Pages
 	page: {
-		title: 'Default (Page)',
-		description: 'Displays the content of a single page',
+		title: __( 'Page' ),
+		description: __( 'Single page template' ),
```

Comment:
```suggestion
		description: __( 'Single-page template' ),
```

Normally it's a stylistic nit pick (as not all styles in English follow this usage of hyphens), but in the context of i18n I believe this will help make the string less ambiguous for translators and therefore less likely to need `_x` or `// translators` :)

Reply:
It makes sense 🤔 
I do have a slight personal preference for the non-hyphenated version, though.

How about "Template for single pages/posts"?
It would also be consistent with the `singular` description ("Default template for both single posts and pages").

(Also, I'm going to update the titles of `page` and `single` to "Single Page" and "Single Post", which was my intention already, but it slipped through the commit)

Reply:
> How about "Template for single pages/posts"?

No strong opinion, that works too. :) Though I'd drop the slash and go with

```suggestion
		description: __( 'Template for single pages and posts' ),
```

Reply:
Ah yeah, sorry, I meant "Template for single pages" _and_ "Template for single posts". 🙂